cdk/BackendStack.ts

```diff
@@ -61,10 +61,10 @@ export class BackendStack extends Stack {
 			value: cd.role.roleArn,
 		})
 		const db = new Timestream.CfnDatabase(this, 'db')
-		const table = new Timestream.CfnTable(this, 'table', {
+		const table = new Timestream.CfnTable(this, 'table2', {
 			databaseName: db.ref,
 			retentionProperties: {
-				MemoryStoreRetentionPeriodInHours: `24`,
+				MemoryStoreRetentionPeriodInHours: `72`,
```

Comment:
**problem:** Testing artifact. Revert.

Reply:
This would make the test fail in CI? Should I make sure the retentionPeriod is 72 hours only in CI?

Reply:
Yes, example: https://github.com/hello-nrfcloud/backend/blob/282cc328e5d2469a3326b0e1e906985b43e39a44/cdk/resources/DeviceStorage.ts#L16